HOW_TO.md

```diff
@@ -0,0 +1,53 @@
+## How to join the network?
+
+RPC address would be provided here shortly.
+
+We recommend you to use [near-cli documentation](https://docs.near.org/tools/near-cli), e.g. how to [create an account](https://docs.near.org/tools/near-cli#near-create-account).
+
+StatelessNet is a sandbox created for load testing purposes.
+The traffic from mainnet may be forwarded to StatelessNet.
+It means that all the existing mainnet accounts would be already occupied.
+It will not affect the activity on mainnet in any case.
+
+If you need some tokens for your activity, fill out the form: [the link to the form will be provided shortly].
+
+## How to become a validator?
+
+The most recent release of StatelessNet: https://github.com/near/nearcore/commit/ef5564c4950d448144d23488df8b63d45e6b0209
+
+You need to build it with the additional argument
+```
+--features statelessnet_protocol
+```
+
+Sample config: [will be provided shortly]
+
+In order to be able to create staking pool, you need some tokens.
+Please fill in the validator form: [the link to the form will be provided shortly].
+
+### Hardware requirements
+
+Assuming the heaviest setup where a node tracks all shards and stores all shards in memory:
+- 1TB disk
+- 32GB RAM
+- 8 cores
+
+We will add the requirements for tracking one shard shortly.
```

Comment:
Shortly is relative, we are not yet ready to enable it (it's not tested). I would say that in later stages of StatelessNet we are planning on enabling single shard tracking and that instructions will be shared then.